scripts/complete_liquidity_provision.js

```diff
@@ -128,6 +129,19 @@ module.exports = async (callback) => {
           }
         })
       )
+      // Detect if provided brackets have existing orders.
+      const existingOrders = await Promise.all(
+        bracketAddresses.map(async (safeAddr) => {
+          return await hasExistingOrders(safeAddr, exchange)
+        })
+      )
+      const dirtyBrackets = bracketAddresses.filter((_, i) => existingOrders[i] == true)
+      if (
+        existingOrders.some((t) => t == true) &&
```

Comment:
Nice syntax! You could even go further and use the identity function:
```suggestion
        existingOrders.some((t) => t) &&
```

Reply:
I don't understand this suggestion... I am assuming that the result of the mapping needs to be a boolean statement? Also, what is the identity function if it isn't `{t => t}`?

Reply:
My understanding is that ` existingOrders.some(f)` returns `true` if `f(x)` is true for at least one `x` in `existingOrders`. In this case `f(x) = (x == true)`, but then you could just write `f(x) = x` to get back `true`.